contrib/kind.yaml.j2

```diff
@@ -1,5 +1,7 @@
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
+featureGates:
+  "MultiCIDRServiceAllocator": true
```

Comment:
what caused you to add this?

Reply:
my intial plan was to enable the serviceCidr feature gate in kind support the feature but it caused lots of issues with dualstack and live migration so I decided to skip the serviceCidr test and say we dont support it for now (its a v1alpha feature) 
I will open an issue noting this

Reply:
yeah eventually when it does graduate to beta we must add support for it. hmm too bad I thought it would have been straightforward.

Reply:
@bpickard22 : qq re https://github.com/ovn-org/ovn-kubernetes/issues/4269
If the feature is enabled by default aren't we supposed to turn off the feature gate? why is this set to true? or does this mean the gate is ON, which means the feature is off?

Reply:
it is not enabled by default, this was part of my attempt to enable it hoping that what needed to be plumbed would happen and everything would be fine, which was not the case, i just didnt drop that change from the commit and instead removed it when i skipped the test